_includes/readmes/cookie-session.md

```diff
@@ -103,6 +103,8 @@ The options can also contain any of the following (for the full list, see
   - `expires`: a `Date` object indicating the cookie's expiration date (expires at the end of session by default).
   - `path`: a string indicating the path of the cookie (`/` by default).
   - `domain`: a string indicating the domain of the cookie (no default).
+  - `partitioned`: a boolean indicating whether to partition the cookie in Chrome for the [CHIPS Update](https://developers.google.com/privacy-sandbox/3pcd/chips) (`false` by default). If this is true, Cookies from embedded sites will be partitioned and only readable from the same top level site from which it was created.
```

Comment:
sp: Boolean is always capitalized.  
"Cookies" should not be capitalized here.
Hyphenate "top-level".